flowmachine/flowmachine/core/server/query_schemas/flows.py

```diff
@@ -32,11 +33,12 @@ class InputToFlowsSchema(OneOfSchema):
 
 
 class FlowsExposed(BaseExposedQuery):
-    def __init__(self, *, from_location, to_location):
+    def __init__(self, *, from_location, to_location, join_type=None):
```

Comment:
You don't need the default `=None` here, since the schema ensures join_type is always present (and defaults to "inner" if not explicitly provided).